docker/registry/alpine/gradle.properties

```diff
@@ -0,0 +1,4 @@
+# Deliberately using old alpine, see issue #1034 in deephaven-core
+# See also alpinelinux/docker-alpine#155
+deephaven.registry.imageName=alpine:3.12
+deephaven.registry.imageId=alpine@sha256:a296b4c6f6ee2b88f095b61e95c7ef4f51ba25598835b4978c9256d8c8ace48a
```

Comment:
Will need to branch this to support more than one sha per tag for os/arch pairs (see https://hub.docker.com/r/envoyproxy/envoy/tags for example, but notice too that if you click on v1.20.0 or whatever, even though it lists two digests, it only takes you to the linux/amd64 variant...)

Reply:
That's not correct - a repo digest (assuming it supports multi arch) will work across any architectures it supports. For example:

```
$ docker buildx imagetools inspect envoyproxy/envoy@sha256:1e4de079793a62a1ed7023bc44228b80d753773dbf24dcf7b318f360ca595b31 
Name:      docker.io/envoyproxy/envoy@sha256:1e4de079793a62a1ed7023bc44228b80d753773dbf24dcf7b318f360ca595b31
MediaType: application/vnd.docker.distribution.manifest.list.v2+json
Digest:    sha256:1e4de079793a62a1ed7023bc44228b80d753773dbf24dcf7b318f360ca595b31
           
Manifests: 
  Name:      docker.io/envoyproxy/envoy@sha256:11cd0b7aefd2d6b5abc2ad9c2937b35dc9b2f10c67e19eca9af780cdc7e9892b
  MediaType: application/vnd.docker.distribution.manifest.v2+json
  Platform:  linux/arm64
             
  Name:      docker.io/envoyproxy/envoy@sha256:e02931a6470916fd1f468d807cded3840062f2effa586ab90b901d7f494cc91f
  MediaType: application/vnd.docker.distribution.manifest.v2+json
  Platform:  linux/amd64
```

```
$docker buildx imagetools inspect alpine@sha256:a296b4c6f6ee2b88f095b61e95c7ef4f51ba25598835b4978c9256d8c8ace48a           
Name:      docker.io/library/alpine@sha256:a296b4c6f6ee2b88f095b61e95c7ef4f51ba25598835b4978c9256d8c8ace48a
MediaType: application/vnd.docker.distribution.manifest.list.v2+json
Digest:    sha256:a296b4c6f6ee2b88f095b61e95c7ef4f51ba25598835b4978c9256d8c8ace48a
           
Manifests: 
  Name:      docker.io/library/alpine@sha256:a721d672be3cdaaf1285c2b60b76d81ab4a8a3cf5a7180a966f2c545305ad6de
  MediaType: application/vnd.docker.distribution.manifest.v2+json
  Platform:  linux/amd64
             
  Name:      docker.io/library/alpine@sha256:b7b182ccc0811af3fc00754c3cc744e45bc3d7d4f6ef9ee47071173f893dbfb0
  MediaType: application/vnd.docker.distribution.manifest.v2+json
  Platform:  linux/arm/v6
             
  Name:      docker.io/library/alpine@sha256:16aea3fc0648559c47f974d7f31096e71b9ec7013224bb29e35cf3db375af976
  MediaType: application/vnd.docker.distribution.manifest.v2+json
  Platform:  linux/arm/v7
             
  Name:      docker.io/library/alpine@sha256:df5a62c11342b503e692ac719f9569da3518c8f1d5a7f072ad76f1b5ef8d3d43
  MediaType: application/vnd.docker.distribution.manifest.v2+json
  Platform:  linux/arm64/v8
             
  Name:      docker.io/library/alpine@sha256:38f483c3c603557ed5f9a287d13dd9ca8daea07435ad3f06b038621a2f9fc1d9
  MediaType: application/vnd.docker.distribution.manifest.v2+json
  Platform:  linux/386
             
  Name:      docker.io/library/alpine@sha256:372de563e36c3989cee334c3cedf72d0e80879ed989ebd8dee42bdbff5fe4184
  MediaType: application/vnd.docker.distribution.manifest.v2+json
  Platform:  linux/ppc64le
             
  Name:      docker.io/library/alpine@sha256:5391bba453c1474d620fd092c96342e09e0a8ee7f1c2e95ae2dc6e3ed44408c9
  MediaType: application/vnd.docker.distribution.manifest.v2+json
  Platform:  linux/s390x
```